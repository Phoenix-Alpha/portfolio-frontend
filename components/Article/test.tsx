import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Article } from '.'

describe('<Article />', () => {
  it('should renders correct <Article />', () => {
    const wrapper = shallow(<Article />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
