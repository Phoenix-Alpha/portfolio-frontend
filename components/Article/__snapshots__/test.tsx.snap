// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Article /> should renders correct <Article /> 1`] = `<article />`;
